src/Analyzers/Core/Analyzers/RemoveUnusedMembers/AbstractRemoveUnusedMembersDiagnosticAnalyzer.cs

```diff
@@ -67,7 +68,18 @@ protected virtual void HandleNamedTypeSymbolStart(SymbolStartAnalysisContext con
         private sealed class CompilationAnalyzer
         {
             private readonly object _gate;
-            private readonly Dictionary<ISymbol, ValueUsageInfo> _symbolValueUsageStateMap;
+            // State map for candidate member symbols, with the value indicating how each symbol is used in executable code.
```

Comment:
`///` doc comment plz (method below as well).

Reply:
Doc comment field? Also the first "method" below is constructor. Is it the right spot where you want this or you misclicked when selecting lines?

Reply:
> Doc comment field? 

Yes. Thanks :-)